pkg/roachprod/install/cluster_synced.go

```diff
@@ -2563,7 +2602,7 @@ func (c *SyncedCluster) pgurls(
 		if err != nil {
 			return nil, err
 		}
-		m[node] = c.NodeURL(host, desc.Port, virtualClusterName, desc.ServiceMode, DefaultAuthMode(), "" /* database */)
```

Comment:
Is there a reason this is being changed to `AuthUserCert`? `DefaultAuthMode` *should* default to AuthUserCert anyway.